contributors/devel/welcome-to-kubernetes-new-developer-guide.md

```diff
@@ -0,0 +1,126 @@
+
+Welcome to Kubernetes! (New Developer Guide)
+============================================
+
+2017-06-21 through 2017-08-06
+Original author: [Alexander Campbell](https://github.com/alexandercampbell)
+
+---
+
+_This document assumes that you know what Kubernetes does. If you don't, check
+out [the Kubernetes keynote](https://www.youtube.com/watch?v=8SvQqZNP6uo) and
+try the demo at [https://k8s.io/](https://k8s.io/)._
+
+Introduction
+------------
+
+Have you ever wanted to contribute to the coolest cloud technology? This
+document will help you understand the organization of the Kubernetes project and
+direct you to the best places to get started. By the end of this doc, you'll be
+able to pick up issues, write code to fix them, and get your work reviewed and
+merged.
```

Comment:
Oh, also, reviewing and linking to https://github.com/kubernetes/community/tree/master/contributors/devel#kubernetes-developer-guide and potentially updating it with this would be helpful. That doc kind of just lists the pertinent docs in this dir for various topics.